apps/ledger-live-mobile/src/components/Storyly/shared.tsx

```diff
@@ -0,0 +1,12 @@
+export const testStoryInstanceID = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY2NfaWQiOjY5NDgsImFwcF9pZCI6MTE0MjIsImluc19pZCI6MTIxOTh9.XqNitheri5VPDqebtA4JFu1VucVOHYlryki2TqCb1DQ"
+export const onboardingTipsStoryInstanceID =
+  "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY2NfaWQiOjY5NDgsImFwcF9pZCI6MTE0MjIsImluc19pZCI6MTI0ODh9.gFt9c5R8rLsnYpZfoBBchKqo9nEJJs5_G3-i215mTlU";
```

Comment:
Do you think we could have thoses in the env files so eventually we can work with multiple envs?

Reply:
The thing is on Storyly's side it's actually impossible to copy/transfer a story for a storyly instance to another so it wouldn't make sense to test them in one environment and then "promote" to another env ...
We could maybe use what they call "segments" for that, we use these already for localization.

Reply:
I need to document all that stuff actually...